### PR TITLE
Ana/fix csp for validators images

### PR DESCRIPTION
**Pull request by Bitcoinera**
Just need to clarify the Keplr and Nuxt errors (or maybe they are the same)

### CODE REVIEW

File: netlify.toml [hunk 1]
```diff
@@ -7,7 +7,7 @@
     Content-Security-Policy = '''
       default-src 'self' https://*.netlify.app;
       connect-src 'self' https://keybase.io https://lcd.nylira.net;
-      img-src 'self' data: https://*.netlify.app;
+      img-src 'self' data: https://*.netlify.app https://s3.amazonaws.com/keybase_processed_uploads;
```
Comment (jbibla) [inline netlify.toml:10]:
```suggestion
      img-src 'self' data: https://*.netlify.app https://s3.amazonaws.com/keybase_processed_uploads/*;
```

Comment (jbibla) [reply netlify.toml:10]: not sure - just guessing

Comment (Bitcoinera) [reply netlify.toml:10]: we can try. Looks good

File: netlify.toml [hunk 2]
```diff
@@ -6,8 +6,8 @@
   [headers.values]
     Content-Security-Policy = '''
       default-src 'self' https://*.netlify.app;
-      connect-src 'self' https://keybase.io https://lcd.nylira.net;
-      img-src 'self' data: https://*.netlify.app;
+      connect-src 'self' https://keybase.io https://lcd.nylira.net data:application/octet-stream;base64,AGFzbQEAAAABqAMwYAJ;
```
Comment (faboweb) [inline netlify.toml:9]: I doubt that this works. the code would change with each keplr version

**Pull request merged**